src/components/interfaces.ts

```diff
@@ -6,6 +6,8 @@ export type FlipContext = "both" | "start" | "end";
 export type Kind = "brand" | "danger" | "info" | "inverse" | "neutral" | "warning" | "success";
 export type Layout = "horizontal" | "vertical" | "grid";
 export type LogicalFlowPosition = "inline-start" | "inline-end" | "block-start" | "block-end";
+export type ModeClass = "calcite-mode-light" | "calcite-mode-dark" | "calcite-mode-auto";
+export type ModeName = "light" | "dark" | "auto";
```

Comment:
Nows the time to decide if "auto" is correct or if we want to make it "device" or something like that 👍

Reply:
Good call, will leave this one up to the masses.

Reply:
"system"?

Reply:
"browser"? This mode pulls from the browser setting right?

Reply:
Inferred usually from a combination of system / device and browser.

Reply:
Its probably ok to leave it as "auto" since there isn't a perfect word for browser/device/system.